test/cpp/interop/server_helper.h

```diff
@@ -43,6 +44,16 @@ namespace testing {
 
 std::shared_ptr<ServerCredentials> CreateInteropServerCredentials();
 
+class InteropContextInspector {
+ public:
+  InteropContextInspector (const ::grpc::ServerContext& context);
```

Comment:
Done. Note that msan complains about context_ not being used. It'll be used alright in a following PR.